test/homework_test/01_variables_test/01_variables_tests.cpp

```diff
@@ -12,3 +12,18 @@ TEST_CASE("Verify sum_numbers function")
 	REQUIRE(add_numbers(10, 10) == 20);
 }
 
+//Verify get_sales_tax_amount with parameter 10 returns the value .675
+//Verify get_sales_tax_amount with parameter 20 returns the value 1.35
+TEST_CASE("Test Get Sales Tax.")
+{
+	REQUIRE(get_sales_tax_amount(10) == 0.675);
+	REQUIRE(get_sales_tax_amount(20) == 1.35);
```

Comment:
get_sales_tax_amount(10) should be get_sales_tax(10)
get_sales_tax_amount(20) should be get_sales_tax(20)